<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1040</width>
    <height>828</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="kcs_mainMenu">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>811</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="mainLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>1024</width>
       <height>50</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Arial'; font-size:20pt; font-weight:600;&quot;&gt;Key Control System&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="main_key1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>200</width>
       <height>150</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="main_key2">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>180</y>
       <width>200</width>
       <height>150</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="main_key3">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>180</y>
       <width>200</width>
       <height>150</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="main_key4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>500</y>
       <width>150</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Meeting Room&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="main_key5">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>490</y>
       <width>150</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;Cargo Door&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="main_key6">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>490</y>
       <width>150</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;System Room&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="main_key_7">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>490</y>
       <width>150</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Vacuum Room&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="main_key4_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="main_key5_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>490</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="main_key6_2">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>490</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="main_key_7_2">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>490</y>
       <width>60</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="main_key1_status">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>330</y>
       <width>200</width>
       <height>47</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#00aa00;&quot;&gt;Available&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="main_key2_status">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>330</y>
       <width>200</width>
       <height>47</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#00aa00;&quot;&gt;Available&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="main_key3_status">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>330</y>
       <width>200</width>
       <height>47</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#00aa00;&quot;&gt;Available&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="main_key4_status">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>540</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#00aa00;&quot;&gt;Available&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="main_key5_status">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>540</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#00aa00;&quot;&gt;Available&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="main_key6_status">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>540</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#00aa00;&quot;&gt;Available&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="main_key7_status">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>540</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#00aa00;&quot;&gt;Available&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="time">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>740</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>999</x>
       <y>740</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>R</string>
     </property>
    </widget>
    <widget class="QLabel" name="main_key9">
     <property name="geometry">
      <rect>
       <x>780</x>
       <y>180</y>
       <width>200</width>
       <height>150</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="main_key9_status">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>330</y>
       <width>200</width>
       <height>47</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#00aa00;&quot;&gt;Available&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="kcs_get_give">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>768</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QToolButton" name="get_key">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>250</y>
       <width>250</width>
       <height>200</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="give_key">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>250</y>
       <width>250</width>
       <height>200</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="kcs_getKey">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>768</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="get_key1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>220</width>
       <height>170</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="get_key2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>230</width>
       <height>170</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="get_key3">
     <property name="geometry">
      <rect>
       <x>515</x>
       <y>30</y>
       <width>230</width>
       <height>170</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="get_key4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>235</y>
       <width>250</width>
       <height>180</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="get_key5">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>235</y>
       <width>250</width>
       <height>180</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="get_key6">
     <property name="geometry">
      <rect>
       <x>725</x>
       <y>235</y>
       <width>250</width>
       <height>180</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="get_key7">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>450</y>
       <width>250</width>
       <height>180</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="get_key8">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>450</y>
       <width>250</width>
       <height>180</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="get_cancel">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>660</y>
       <width>165</width>
       <height>90</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="get_key9">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>30</y>
       <width>230</width>
       <height>170</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="kcs_giveKey">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>768</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QToolButton" name="give_key5">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>235</y>
       <width>250</width>
       <height>180</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="give_key6">
     <property name="geometry">
      <rect>
       <x>725</x>
       <y>235</y>
       <width>250</width>
       <height>180</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="give_key2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>30</y>
       <width>230</width>
       <height>170</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="give_key8">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>450</y>
       <width>250</width>
       <height>180</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="give_key4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>235</y>
       <width>250</width>
       <height>180</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="give_key3">
     <property name="geometry">
      <rect>
       <x>515</x>
       <y>30</y>
       <width>230</width>
       <height>170</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="give_key7">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>450</y>
       <width>250</width>
       <height>180</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="give_key1_">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>220</width>
       <height>170</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="give_cancel">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>660</y>
       <width>165</width>
       <height>90</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="give_key9">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>30</y>
       <width>230</width>
       <height>170</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="kcs_message">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>768</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="message">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>250</y>
       <width>761</width>
       <height>260</height>
      </rect>
     </property>
     <property name="text">
      <string>get or give key</string>
     </property>
    </widget>
    <widget class="QLabel" name="unlock_gif">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>250</width>
       <height>260</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <zorder>kcs_get_give</zorder>
   <zorder>kcs_message</zorder>
   <zorder>kcs_getKey</zorder>
   <zorder>kcs_giveKey</zorder>
   <zorder>kcs_mainMenu</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
